### ISSUE TITLE
Is it necessary to have the `Default` suffix on output variables like `SshCellDefault`?

### ISSUE THREAD

**Issue by xylar** [E3SM-Project/Omega#178]
This feels awkward to me because the suffix `Default` has no context in most output files.

**Comment by philipwjones** [comment]
Yes, I noticed this too and for most other classes (eg State, Tracer) we avoid adding the default suffix so we should be consistent here. Personally, I also think it's unnecessary to have mesh location (Cell, Edge, Vertex) in the name when it is the default location, but that opinion may not be universally held and it could still be useful for some fields where it might not be obvious.

**Comment by xylar** [comment]
I would also prefer to just call the variable `Ssh`.

**Comment by xylar** [comment]
This has been fixed -- the variable is now `SshCell`.
